test/test-watcher-cross-stop.c

```diff
@@ -59,6 +59,10 @@ static void close_cb(uv_handle_t* handle) {
 
 
 TEST_IMPL(watcher_cross_stop) {
+#if defined(__MVS__)
+  RETURN_SKIP("IBM WORK ITEM: 130119");
```

Comment:
I am going to remove this snippet and disable the test case in test/test-list.h for this platform.